typescript/src/includes/calculator.html

```diff
@@ -10,23 +10,23 @@ <h4>Predict metal occupancies in vivo</h4>
   <thead>
     <tr>
       <td></td>
-      <th>Metal affinities of molecule (M)<br/>(dissociation constants)</th>
-      <th>∆G for complexation with the molecule (kJ mol-1)</th>
-      <th>Buffered metal concentration (M)</th>
-      <th>Intracellular available ∆G (kJ mol-1)</th>
-      <th>Molecule occupancy with metal</th>
+      <th>Metal Affinity<br>(M)</th>
+      <th>∆G(kJ mol<sup>-1</sup>)</th>
+      <th>Metal Availability<br>(M)</th>
+      <th>Available ∆G<br>(kJ mol<sup>-1</sup>)</th>
+      <th>Occupancy</th>
     </tr>
   </thead>
   <tbody>
 
   </tbody>
   <tfoot>
-    <th>Total Metalation</th>
-    <td></td>
-    <td></td>
-    <td></td>
-    <td></td>
-    <th id="total_metalation" class="result"></th>
+    <th class="bottom">Total Metalation</th>
+    <td class="bottom"></td>
+    <td class="bottom"></td>
+    <td class="bottom"></td>
+    <td class="bottom"></td>
+    <th id="total_metalation" class="result bottom"></th>
```

Comment:
Given these cells are in `<tfoot>` could you use that instead of adding the `bottom` class to every cell?

Reply:
When I used tfoot it colours the whole table for some reason,

Reply:
Ah...Looks like my javascript is adding the metal rows to `tfoot` not `tbody` - I'll log that as a bug and take a look at some point. Stick with `class="bottom"` for now and then we can replace it with `tfoot` later on.